.github/workflows/release-asset-audit.py

```diff
@@ -34,7 +34,6 @@ def main():
             "zmodem",
             "androm3da",
             "tru",
-            "rovka",
```

Comment:
This is going to cause the audit to fail.  There needs to be logic added that will only allow release managers and the llvmbot account starting with LLVM19.

Reply:
I assumed that this script was branched, but I also forgot that we changed how direct uploads work.

So to be clear, 19 and prior, it was this list of contributors. After 19, it's the two release managers and llvmbot. Anything else is linked from the third party binaries thread on Discourse. Correct?

I can implement that.

Reply:
Yes, that's correct.